similarity.js

```diff
@@ -0,0 +1,20 @@
+#! /usr/bin/env node
+
+//const distance = require('leven');
```

Comment:
Thanks for the PR!

Looks good. I would just remove this comment.

Also it would be good to add a test for the CLI. I've enjoyed using http://ghub.io/nixt for that. Are you up for it?